### ISSUE TITLE
Collect stats from yii-demo

### ISSUE THREAD

**Issue by samdark** [yiisoft/di#206]
1. How many services are requested per route?
2. How many aren't defined in container and automatically resolved?
3. How many sub-dependencies are per service?

**Comment by yiiliveext** [comment]
> 1. How many services are requested per route?
> 2. How many aren't defined in container and automatically resolved?
> 3. How many sub-dependencies are per service?

This easy imlements with yii-debug.

**Comment by samdark** [comment]
We need to get these numbers once to better understand what may affect performance and what may not.

**Comment by alamagus** [comment]
Any thoughts on having built-in support for opentelemetry, like, yiisoft/otel?
https://github.com/open-telemetry/opentelemetry-php (it's not production ready yet though)
https://groups.google.com/g/php-fig/c/6Nm0ZQ1_hJA/m/Fxp6EPsODgAJ

**Comment by samdark** [comment]
@alamagus I don't think we are going to support it. Instead, I'd rely on community solutions for it.

**Comment by xepozz** [comment]
> Any thoughts on having built-in support for opentelemetry, like, yiisoft/yii-otel?
> https://github.com/open-telemetry/opentelemetry-php (it's not production ready yet though)
> https://groups.google.com/g/php-fig/c/6Nm0ZQ1_hJA/m/Fxp6EPsODgAJ

@alamagus Thanks for the information, but I think this issue is not about telemetry in general, but about some benchmarks in CI.
If you want to add `Open Telemetry` integration it would be nice to have as separated package that could be included into users apps. 
Both `yiisoft/demo` and `yiisoft/demo-api` repos may be lab rats for these changes. Would you like to implement it?
